rclpy/src/rclpy/_rclpy.c

```diff
@@ -2769,7 +2770,7 @@ rclpy_get_ready_entities(PyObject * Py_UNUSED(self), PyObject * args)
     GET_LIST_READY_ENTITIES(guard_condition)
   }
   Py_DECREF(entity_ready_list);
-  PyErr_Format(PyExc_RuntimeError,
+  PyErr_Format(RCLError,
```

Comment:
Not an error from `rcl`.
```suggestion
  PyErr_Format(PyExc_RuntimeError,
```